broker-util/oo-admin-upgrade

```diff
@@ -490,7 +490,7 @@ class Upgrader
       inactive_gears = []
 
       gears.each do |gear|
-        if gear_whitelist && gear_whitelist.length > 0
+        if !gear_whitelist.include?(gear[:uuid]) && gear_whitelist.length > 0
           puts "Gear #{gear[:uuid]} is not in the whitelist and will be skipped"
           next unless gear_whitelist.include?(gear[:uuid])
```

Comment:
Thanks! There does appear to be a logic error here, but I think we should be leaving the `if` condition the same (because we need to check whether `gear_whitelist` is `nil`) and changing its body from

```ruby
           puts "Gear #{gear[:uuid]} is not in the whitelist and will be skipped"
           next unless gear_whitelist.include?(gear[:uuid])
```

to


```ruby
           unless gear_whitelist.include?(gear[:uuid])
             puts "Gear #{gear[:uuid]} is not in the whitelist and will be skipped"
             next
           end
```

(because we should only print the "will be skipped" message if we actually skip it with `next`).

Does that make sense to you, and if so, would you mind updating the PR please?